### ISSUE TITLE
Crashes at:

### ISSUE THREAD

**Issue by haroon47** [Lauszus/FaceRecognitionApp#39]
java.lang.UnsatisfiedLinkError: dlopen failed: library "../../lib/libtbb.so" not found

**Comment by Lauszus** [comment]
@haroon47 sorry, but that library should not be needed to run this app. Have you modified the code in any way?

**Comment by haroon47** [comment]
I did not change anything. I tried to figure it out for a week but to no avail. Maybe there is problem of NDK integration at my side.

**Comment by Lauszus** [comment]
@haroon47 please try to use the exact version used by the build script, as that was has been used to build the APK and what I have used for testing: https://github.com/Lauszus/FaceRecognitionApp/blob/master/.travis.yml#L49-L51.

**Comment by jamenenedez** [comment]
Hi, I am using the app for school work and I needed to add another functionality. This functionality is for web scrapping. I was using eigen face component, and it works fine, but when I decided to use fisher face component the app crash just when I return from the activity I added. The crash message was: 

"A/libc: app/src/main/cpp/FaceRecognitionLib/LDA.cpp:45: int32_t LDA::compute(const Eigen::MatrixXf &, const Eigen::VectorXi &, int32_t): assertion "(size_t)classes.size() == N" failed
Fatal signal 6 (SIGABRT), code -6 (SI_TKILL) in tid 3583 (AsyncTask #3), pid 2845 (erecognitionapp)"
